### ISSUE TITLE
TexStudio crashing (?)

### ISSUE THREAD

**Issue by playingmoz** [texstudio-org/texstudio#5]
A pop-up states TexStudio CRASHED due to SIGSEGV. Any insight?

**Comment by sunderme** [comment]
what OS ? what version of txs ?
What were you doing, when txs crashed ? Can you reproduce the crash ?
And there should be a backtrace.txt somewhere.

**Comment by sunderme** [comment]
no feedback

**Comment by playingmoz** [comment]
Hi, sorry for the delay. I’m using MacOS, High Sierra, (with update) on TeXstudio’s latest version with MacTeX 2017. 

It often happens when I’m compiling documents of large size with many input files. For example, I wrote a textbook with 11 chapters, all inputs in the parent file. The issue is with TeXStudio, I figured out, with the backtrace.txt and when I try to recover it, it crashes immediately with the backtrace but with _1 after it. :-/ I end up closing it and reopening it. I hope this helps. I just wondered if there was any insight to this error. Thanks!
